sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/table.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.catalyst.analysis
+
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.catalyst.plans.logical.LeafNode
+import org.apache.spark.sql.connector.catalog.{Identifier, Table, TableCatalog}
+
+case class ResolvedTable(catalog: TableCatalog, identifier: Identifier, table: Table)
```

Comment:
I think it's better to use `table: NamedRelation`, to be consistent with the existing commands like `DescribeTable`.

When we add `ResolveView` later, we would also use `view: NamedRelation`, as it can be temp views.

Reply:
So we need to add support for v1 table to be converted to `NamedRelation`. https://github.com/apache/spark/pull/26847/files#diff-57b3d87be744b7d79a9beacf8e5e5eb2R883-R888

Reply:
yup